selectors/drafts.ts

```diff
@@ -54,12 +54,14 @@ export function makeGetDraftsByPrefix(prefix: string): DraftSelector {
 
             return Object.keys(storage).flatMap((key) => {
                 const item = storage[key];
+                const isCorrectDraftType = includeLegacyDrafts ? typeof item.value?.channelId === 'undefined' : item.value?.show;
```

Comment:
using `typeof` seems a bit overkill. Normally using the value in itself should be enough when checking for a non-strict falsey, but when you need to make sure to have a `boolean` value just use `Boolean()`. 

```suggestion
                const isCorrectDraftType = includeLegacyDrafts ? Boolean(item.value?.channelId) : item.value?.show;
```